shared/gh/js/utils/gh.utils.templates.js

```diff
@@ -139,4 +139,40 @@ define(['exports', 'gh.constants'], function(exports, constants) {
         // Always return the rendered HTML string
         return compiled;
     };
+
+    /**
+     * Render the hierarchy string
+     *
+     * @param  {Object}    orgUnit      The organisational unit to start building the hierarchy structure with
+     * @param  {String}    separator    The string used to split the organisational units
+     * @return {String}                 The generated hierarchy string
+     */
+    var renderHierarchyString = exports.renderHierarchyString = function(orgUnit, separator) {
```

Comment:
Will we be using this in other areas as well? If not, I'd move this over to the modal.